### DISCUSSION TITLE
Manually setting Rout Priority on Known Nodes

### DISCUSSION THREAD

**Discussion by SimbimChimbetov** [meshtastic/Discussions#32] [Firmware]
Is a Funktion possible to manuelly set the Rout Priority one step up or down on known Nodes?

Here is the Problem: I Installed a Router in my City at the higest Spot and one at the top of a Mountain 15 km away, this opened a huge Area and a lot of Nodes. We have a Mesh now, with approx 70 km range and 100 Nodes.

Now some Nodes in my City and the other Cities in range have their Nodes placed on the Roof of their Houses in "ROUTER" Role and I can watch the Tracerout hop from my City to my Router on the Mountain and then back to my City, at the end wasting one to two hops just to leave my City.

I regularly try to communicate and explain over the Mesh when to use the "Router Role", also Posted the Youtube Video ["Meshtastic Routing Issues & Deployment Scenarios"](https://www.youtube.com/watch?v=htjwtnjQkkE)
Many users switched to "CLIENT_MUTE" but still we have some Nodes who knows better and don't care about Roles, max hops and Intervals.

This is why I wish there was a Funktion on my "ROUTER" Nodes to manually set the priority on known nodes down to priority of a "CLIENT" or up.

**Comment by ikearat** [comment]
What happens if you use "Ignore" on the nodes you are not interested in?

**Reply by SimbimChimbetov** [reply]
I could ignore max 3 Nodes Incoming only but my Router will still route via that Node and waste hops.
Also If I Add them to the Ignore List, they will be blind because the only way out of their City is my Router on the Mountain.

**Reply by ikearat** [reply]
Thank you for following up on my post. I am smarter now! And thank you for the consideration of those user's nodes.
There is a single "Router" node visible to me and it is not Internet (MQTT) connected or is blocking that traffic. I can "do" MQTT myself but with only my node doing that it's kinda a waste of time. I'm not trying to do "nets" like the hams do or looking for long range contacts. When I first started there was a node that actually responded to a message but they have gone silent and the router is aparently unmonitored. It's dark here under the porch :)
